index.html

```diff
@@ -859,194 +859,180 @@
-its user present the [=identity=] they want in each [=context=] they find themself
-in.</span>
-
-Sometimes this means the UA should ensure that one site can't learn anything about their
-user's behavior on another site, while at other times the UA should help their user prove to
-one site that they have a particular identity on another site.
+<span class="practicelab" id="principle-identity-per-context">A [=user agent=]
+should help its user present the [=identity=] they want in each [=context=]
+they are in.</span>
 
 </div>
 
-To do this, [=user agents=] have to make some assumptions about the borders between [=contexts=]. By
-default, [=user agents=] define a <dfn>machine-enforceable context</dfn> or <dfn>partition</dfn> as:
-
-* A set of [=environments=] (roughly iframes (including cross-site iframes), workers, and top-level
-  pages)
-* whose [=environment/top-level origins=] are in the [=same site=] (but see [[PSL-Problems]])
-* being visited within the same user agent installation (and browser profile, container, or
-  container tab for user agents that support those features)
-* between points in time that the person or user agent clears that [=site=]'s cookies and other
-  storage (which is sometimes automatic at the end of each session).
-
-Even though this is the default, [=user agents=] are free to restrict this context as
-people need. For example, some user agents may help people present different
+A [=user agent=] which upholds this principle will sometimes ensure that *one*
+site can't learn anything about its user's behavior on *another* site. At
+other times, the UA may help its user *prove* to one site that they have a
+particular identity on another site. Similarly, a UA will help its user to
+separate or communicate identity across *repeat* visits to the *same* site.
+
+A [=person=]'s <dfn>identity</dfn> is the set of characteristics that define
+them. Their identity *in a [=context=]* is the set of characteristics they
+present in that context. People frequently present different identities to
+different contexts, and also frequently share an identity among several
+contexts.
+
+In computer systems and on the web, an [=identity=] seen by a particular
+website is typically assigned an <dfn>identifier</dfn> of some type, which
+makes it easier for an automated system to store data about that [=person=].
+Examples of [=identifiers=] for a [=person=] can be:
+* their name,
+* an identification number including those mapping to a device that this
+[=person=] may be using,
+* their phone number,
+* their location data,
+* an online identifier such as email or IP addresses,
+* browser [=fingerprinting|fingerprints=] (based on a combination of
+configuration characteristics), or
+* factors specific to their physical, physiological, genetic, mental,
+economic,
+cultural, social, or behavioral [=identity=],
+* strings derived from [=identifiers=], for instance through hashing.
```

Comment:
This block isn't really needed for the flow of this section, even though we use the "identifier" definition elsewhere. Maybe we should move it to the Common Concepts appendix?